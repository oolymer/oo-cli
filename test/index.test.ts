import { analyze } from ".."
import { toMatchImageSnapshot } from "jest-image-snapshot"
import * as fs from "fs"

expect.extend({ toMatchImageSnapshot })

describe("oo", () => {

  it("should analyze fixtures", async () => {
    expect(await analyze("test/__fixtures__")).toMatchSnapshot()
  })

  it("should match images", () => {
    const image = fs.readFileSync("test/__fixtures__/4a.png")
    expect(image).toMatchImageSnapshot()
  })

})
