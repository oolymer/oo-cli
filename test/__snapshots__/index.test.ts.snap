// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`oo should analyze fixtures 1`] = `
"## FooPage

> element defined in foo-page.html)

A custom element with **properties** and **methods**.

\`\`\`html
<foo-page></foo-page>
\`\`\`

## Foo.Mixin

> mixin defined in foo-mixin.html)

A custom element class mixin that provides a collection of utility methods.
"
`;
